frontend/src/styles/style.css

```diff
@@ -8,3 +8,18 @@
   --SLURP-OFFWHITE: #FFFDFA;
   --SLURP-GRAY: #545352;
 }
+
+/* Set dynamic content to use the entirety of the window. */
+html, body, #root, #root>div, #root>div>* {
+  height: 100%
+}
+
+/* Pages with the hero image spans the entire window with a slight dark tint. */
+.hero-image {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.1)),
```

Comment:
Can you write in a `/*comment*/` what this does?

Reply:
Added. 👍